addons/spreadsheet/static/tests/lists/list_plugin.test.js

```diff
@@ -394,6 +397,43 @@ test("Can undo/redo a delete list", async function () {
     expect(B4.value).toBe("#ERROR");
 });
 
+test("can update a list", async () => {
```

Comment:
shouldn't we add a test that observe that the RPC are done according to the new configuration ?
It would have also caught the double web_search_read call.